nixos/modules/services/networking/twingate.nix

```diff
@@ -14,7 +14,7 @@ in {
   config = mkIf cfg.enable {
 
     networking.firewall.checkReversePath = lib.mkDefault false;
-    networking.networkmanager.enable = true;
```

Comment:
Just wanted to note: This may still be required. I tried using this derivation as a patch in my personal dotfiles and `twingated` seems to fail when NetworkManager isn't there. I also ran into a non-related with issue with NetworkManager 1.42.x which seems to be related to this possibly: https://forum.twingate.com/t/error-libsdwan-configure-sdwan-dns-failed-to-add/212